discord/colour.py

```diff
@@ -226,5 +226,13 @@ def blurple(cls):
     def greyple(cls):
         """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
         return cls(0x99aab5)
+    
+    @classmethod
+    def darker_greyple(cls):
+        """A factory method that returns a :class:`Colour` with a value of ``0x36393F``.
+        Which is the value of discords dark-theme background."""
+        return cls(0x36393F)
+
+Colour.transparent = Colour.darker_greyple
```

Comment:
Aliasing this to `transparent` is misleading. It is not *actually* transparent, having shading differences on mobile and not working at all on light theme. We should not make the assumption that users are using dark theme, especially considering that light theme is still the default theme for Discord, but even so it is inaccurate.